src/components/DevToolsDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Dialog, Flex, Switch, Text } from '@radix-ui/themes'
+import useKonami from 'use-konami'
+import { useState } from 'react'
+
+import { useFeaturesStore } from '@/store/features'
+import { Feature } from '@/types/features'
+
+export function DevToolsDialog() {
```

Comment:
If we want to leave this out in production builds, then we could do this:

```tsx
function DevModeDialog() {
  // Full implementation
}

function ProdModeDialog() {
  return null
}

export const DevToolsDialog = 
  import.meta.env.MODE === "development" 
    ? DevModeDialog 
    : ProdModeDialog
```

Vite will replace `import.meta.env.MODE` with the static value so that e.g. a production build will have `"production" === "development"` . Then tree shaking recognizes that `"production" === "development"` will always be false, removes the truthy branch and, since it's no longer referenced anywhere, the `DevModeDialog` as well.

Reply:
Good suggestion, thanks!